util/src/entities/MemberVerificationForm.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, Entity, Index, JoinColumn, ManyToOne, PrimaryColumn, RelationId } from "typeorm";
+import { Guild } from "./Guild";
+import { BaseClassWithoutId } from "./BaseClass";
+
+export interface MemberVerificationFormField {
+	field_type: string;
+	label: string;
+	required: boolean;
+	values?: string[]; // for rules
+	choices?: string[]; // for multiple choice
+	placeholder?: string; // for short answers and paragraphs
+}
+
+@Entity("member_verification_forms")
+@Index(["id"], { unique: true })
+export class MemberVerificationForm extends BaseClassWithoutId {
+	@PrimaryColumn()
+	@RelationId((form: MemberVerificationForm) => form.guild)
+	id: string;
+
+	@JoinColumn({ name: "id" })
+	@ManyToOne(() => Guild, {
+		onDelete: "CASCADE",
+	})
+	guild: Guild;
```

Comment:
for the possibility to use a questionnaire in many guilds, guild should be added into the response object at response time and the guild row should include the questionnaire id